### PR TITLE
Update various parts of the readme

### PR DESCRIPTION
**Pull request by farmdawgnation**
Various pieces of our readme were super out of date, so I updated them.

Someone check me on the proposed changes to the build file, please. :)

### CODE REVIEW

**Comment by farmdawgnation** [comment]
Huh, I made another commit on this branch and it's not showing up...

File: README.md [hunk 1]
```diff
@@ -138,16 +139,15 @@ The Lift wiki is hosted on Assembla and can be found at [http://www.assembla.com
 
 ### ScalaDocs
 
-The ScalaDocs for each release of Lift, in additional to the actual JARs, are available on the Liftweb.net site. You can access the source code–based documentation for releases via the site's homepage or by navigating directly to the URL for the specific release. For instance, the Lift 2.5 release can be accessed at [http://liftweb.net/api/25/api/](http://liftweb.net/api/25/api/).
+The ScalaDocs for each release of Lift, in additional to the actual JARs, are available on the Liftweb.net site. You can access the source code–based documentation for releases via the site's homepage or by navigating directly to the URL for the specific release. For instance, the Lift 3.0 release can be accessed at [http://liftweb.net/api/25/api/](http://liftweb.net/api/30/api/).
```
Comment (Shadowfiend) [inline README.md]: Whoops, the URL should be `/30/api` inside the link as well, I believe.

File: README.md [hunk 2]
```diff
@@ -68,17 +68,18 @@ Or, you can add Lift to your `pom.xml` like so:
 
     <dependency>
       <groupId>net.liftweb</groupId>
-      <artifactId>lift-mapper_${scala.version}</artifactId>
-      <version>2.5.1</version>
+      <artifactId>lift-webkit_${scala.version}</artifactId>
+      <version>3.0.1</version>
```
Comment (Shadowfiend) [inline README.md]: Given that we don't have any reports of serious issues, we could go ahead and just set this to `3.1.0` (here and in the above change as well) and then merge it when the 3.1.0 release goes out.

Comment (farmdawgnation) [inline README.md]: I'm good with that.

File: README.md (hunk 1 above)
Comment (farmdawgnation) [inline README.md]: Lawl yep.

**Pull request merged**